### PR TITLE
[TS] LPS-194839 

### PR DESCRIPTION
**Pull request by MirnaGama**
For steps to reproduce and implementation details, please check: https://github.com/liferay-appsec/liferay-portal/pull/1343

-  LPS-194839 Avoid setting user as guest user and check if user is active and not impersonated instead
- LPS-194839 If the user is not active when remember me is marked then do automatically remove cookie so the error message is not looped in menu area
- LPS-194839 Remove impersonation if the value of the doAsUserId parameter is from an inactive user
- LPS-194839 Remove the impersonate option from the actions menu of an inactive user

### CODE REVIEW

**Comment by MirnaGama** [comment]
ci:test:relevant

**Comment by MirnaGama** [comment]
ci:test:security

File: modules/apps/users-admin/users-admin-web/src/main/java/com/liferay/users/admin/web/internal/servlet/taglib/util/UserActionDropdownItems.java [hunk 1]
```diff
@@ -106,7 +106,7 @@ public List<DropdownItem> getActionDropdownItems() {
 						!PropsValues.PORTAL_JAAS_ENABLE &&
 						PropsValues.PORTAL_IMPERSONATION_ENABLE &&
 						(_user.getUserId() != _themeDisplay.getUserId()) &&
-						!_themeDisplay.isImpersonated() &&
+						_user.isActive() && !_themeDisplay.isImpersonated() &&
```
Comment (stian-sigvartsen) [inline modules/apps/users-admin/users-admin-web/src/main/java/com/liferay/users/admin/web/internal/servlet/taglib/util/UserActionDropdownItems.java:109]: We need to alphabetically sort these conditions. If you just move it in front of `!_themeDisplay.isImpersonated()` that should do it.

**Review by stian-sigvartsen — APPROVED** [review]
Nice job @MirnaGama . Please see my comment about the sorting. Can you send a commit? Apart from that it is good to go!

File: modules/apps/users-admin/users-admin-web/src/main/java/com/liferay/users/admin/web/internal/servlet/taglib/util/UserActionDropdownItems.java (hunk 1 above)
Comment (MirnaGama) [reply modules/apps/users-admin/users-admin-web/src/main/java/com/liferay/users/admin/web/internal/servlet/taglib/util/UserActionDropdownItems.java:109]: Hi @stian-sigvartsen 

It's already in front of `!_themeDisplay.isImpersonated()`, isn't it?

Comment (stian-sigvartsen) [reply modules/apps/users-admin/users-admin-web/src/main/java/com/liferay/users/admin/web/internal/servlet/taglib/util/UserActionDropdownItems.java:109]: Sorry, my bad! Perfect then!

**Comment by stian-sigvartsen** [comment]
ci:test:stable

**Comment by stian-sigvartsen** [comment]
Stable test appears to have failed because of an OAuth 2 interaction with Google APIs. Re-running

**Comment by stian-sigvartsen** [comment]
ci:forward:force

**Comment by stian-sigvartsen** [comment]
ci:forward:force

**Pull request closed without merging**